src/renderer/components/Carousel/index.js

```diff
@@ -143,21 +195,41 @@ const Carousel = ({
   }, [index, slides.length]);
 
   const onDismiss = useCallback(() => {
-    dispatch(setDismissedCarousel(true));
-  }, [dispatch]);
+    setWantToDismiss(true);
+  }, []);
```

Comment:
are you sure you don't want to add `setWantToDismiss` to the dependencies of `useCallback` ?

Reply:
Pretty sure, according to the [faq on hooks](https://reactjs.org/docs/hooks-faq.html#what-can-i-do-if-my-effect-dependencies-change-too-often) referring to a `useState` setter function's presence in the dependencies of a `useCallback`:

> _(The identity of the setCount function is guaranteed to be stable so it’s safe to omit.)_